### PR TITLE
Update remaining references to Jira to GH

### PR DESCRIPTION
**Pull request by damccorm**
Part of https://github.com/apache/beam/issues/21812

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] [**Choose reviewer(s)**](https://beam.apache.org/contribute/#make-your-change) and mention them in a comment (`R: @username`).
 - [x] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [x] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/#make-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

**Comment by damccorm** [comment]
Run Kotlin_Examples PreCommit

**Comment by damccorm** [comment]
Run Java_PVR_Flink_Docker PreCommit

**Comment by damccorm** [comment]
Run SQL PreCommit

**Comment by damccorm** [comment]
Run Java PreCommit

File: examples/notebooks/documentation/transforms/python/elementwise/pardo-py.ipynb [hunk 1]
```diff
@@ -23,7 +23,10 @@
    "execution_count": null,
    "metadata": {
     "cellView": "form",
-    "id": "_-code"
+    "id": "_-code",
+    "vscode": {
+     "languageId": "python"
+    }
```
Comment (TheNeuralBit) [inline examples/notebooks/documentation/transforms/python/elementwise/pardo-py.ipynb:29]: are these changes intentional?

Comment (TheNeuralBit) [reply examples/notebooks/documentation/transforms/python/elementwise/pardo-py.ipynb:29]: Ah I guess vscode added them when it was used for find/replace. I suppose that's harmless.

File: website/www/site/content/en/documentation/programming-guide.md [hunk 2]
```diff
@@ -6892,7 +6892,7 @@ Currently Python external transforms are limited to dependencies available in th
 #### 13.1.3. Creating cross-language Go transforms
 
 Go currently does not support creating cross-language transforms, only using cross-language
-transforms from other languages; see more at [Issue 21767](https://github.com/apache/beam/issues/21767).
+transforms from other languages; see more at [https://github.com/apache/beam/issues/21767](https://github.com/apache/beam/issues/21767).
```
Comment (TheNeuralBit) [inline website/www/site/content/en/documentation/programming-guide.md:6895]: I think the existing one looks fine as-is, should we be changing it?

File: website/www/site/content/en/documentation/programming-guide.md [hunk 3]
```diff
@@ -2193,7 +2193,7 @@ Timers and States are explained in more detail in the
 
 {{< paragraph class="language-go" >}}
 **Timer and State:**
-This feature isn't implemented in the Go SDK; see more at [Issue 20510](https://github.com/apache/beam/issues/20510). Once implemented, user defined Timer and State parameters can be used in a stateful DoFn.
```
Comment (TheNeuralBit) [inline website/www/site/content/en/documentation/programming-guide.md:2196]: Same here and above

File: website/www/site/content/en/documentation/programming-guide.md [hunk 4]
```diff
@@ -5436,7 +5436,7 @@ _ = (p | 'Read per user' >> ReadPerUser()
 {{< /highlight >}}
 
 {{< highlight go >}}
-This is not supported yet, see BEAM-10660.
+This is not supported yet, see https://github.com/apache/beam/issues/20510.
```
Comment (TheNeuralBit) [inline website/www/site/content/en/documentation/programming-guide.md:5439]: Should these be `[Issue 20510][https://github.com/apache/beam/issues/20510]`?

**Review by TheNeuralBit — COMMENTED** [review]
Note I don't believe this one has any pressure to get in for the release cut. The only publicly visible changes look to be in website pages, and the website is continuously deployed, separately from the release.

File: examples/notebooks/documentation/transforms/python/elementwise/pardo-py.ipynb (hunk 1 above)
Comment (damccorm) [reply examples/notebooks/documentation/transforms/python/elementwise/pardo-py.ipynb]: Yeah they probably are, but I removed them anyways since they're not helpful

File: website/www/site/content/en/documentation/programming-guide.md (hunk 4 above)
Comment (damccorm) [reply website/www/site/content/en/documentation/programming-guide.md:5439]: No - this is rendered inside a code block so it would render the full markdown I think

File: website/www/site/content/en/documentation/programming-guide.md (hunk 2 above)
Comment (damccorm) [reply website/www/site/content/en/documentation/programming-guide.md]: Nope, fixed!

File: website/www/site/content/en/documentation/programming-guide.md (hunk 3 above)
Comment (damccorm) [reply website/www/site/content/en/documentation/programming-guide.md]: Fixed!

**Comment by damccorm** [comment]
Run Python_PVR_Flink PreCommit

**Comment by damccorm** [comment]
Run PythonDocker PreCommit

**Comment by damccorm** [comment]
Run Portable_Python PreCommit

**Comment by damccorm** [comment]
Run Java PreCommit

**Comment by damccorm** [comment]
Run GoPortable PreCommit

**Comment by damccorm** [comment]
Run GoPortable PreCommit

**Comment by damccorm** [comment]
Run Python_PVR_Flink PreCommit

**Comment by damccorm** [comment]
Run PythonDocker PreCommit

**Comment by damccorm** [comment]
Run Portable_Python PreCommit

**Comment by damccorm** [comment]
Run Java PreCommit

**Comment by damccorm** [comment]
Run Java PreCommit

**Comment by damccorm** [comment]
Run Go PreCommit

File: website/www/site/content/en/documentation/programming-guide.md (hunk 4 above)
Comment (TheNeuralBit) [reply website/www/site/content/en/documentation/programming-guide.md:5439]: Ah good point, thank you
